firmware/integration/rusefi_config.txt

```diff
@@ -858,7 +858,7 @@ custom pin_mode_e 1 bits,    U08,   @OFFSET@, [0:6], @@pin_mode_e_enum@@
 	float[CLT_CURVE_SIZE] cltIdleRpmBins;CLT-based target RPM for automatic idle controller;"C",        1,     0,   -100.0,    250.0,  2
 	float[CLT_CURVE_SIZE] cltIdleRpm;See idleRpmPid;"",      1,     0,      0.0,    8000.0,   0
 	
-	float targetVBatt;set targetvbatt VOLTS;"Volts",      1,      0,       0,30,      1
+	float targetVBatt;+This it the target battery voltage the alternator PID control will attempt to maintain;set targetvbatt VOLTS;"Volts",      1,      0,       0,30,      1
```

Comment:
"This **is** the target..." ?

Reply:
Sounds like proper English to me...

Reply:
Good spot, thanks for casting a second set of eyes over this as that kind of typo is not something a spell checker will pick up. Corrected and will do another PR.